### PR TITLE
Skeleton activities, somewhat working MainActivity

### LINKED ISSUE #1: US 01.01.01
As an owner, I want to publish an experiment with a description, a region, and a minimum number of trials.

# Acceptance Criteria
- Have a button for creating a new experiment from the main activity.

- Button launches a fragment with the following fields:
    - Experiment description (TextEdit)
        - Unlimited length
    - A dropdown with Experiment types (Counts Binomial, Non-negative Count, Measurement)
    - Region (a text field to specify the general location where trials are expected to be conducted)
    - Minimum number of trials (used to indicate when an experiment can be ended)

- The fragment submits data for the experiment
    - If the data is valid, a new experiment is published and shown on the main activity.
    - Otherwise, an error message is shown.

- Have a storyboard demonstrating how a new experiment is created.
- Wiki entry
- UML for frontend components

- Link to relevant UML diagrams (#25) 

# Unknowns
- None

### LINKED ISSUE #32: US 01.01.03
As a developer, I would like files created for each activity and fragment in the application.

# Acceptance criteria
- Skeleton XML and Java files created for each activity and fragment shown in the Figma storyboards.

### PR DESCRIPTION
**Pull request by n30phyte**
Closes #32, partially addresses #1

Will try to complete #1 by end of day today in a new PR

### CODE REVIEW

**Review by PimpinPenguin — APPROVED** [review]
There are a couple differences from the storyboard:
  - The icons for explore and subscription
  - The colour of the bottom navbar
  - The position of the sync button
 
I don't disagree with the changes but at some point one or the other should be updated.

Otherwise lgtm.

**Comment by n30phyte** [comment]
I went for the path of least resistance/cleanest code for all those choices. I figured least code that would have the closest layout was a good way to go about it.

For the Sync button specifically, Google makes it so it's hamburger or fight it for a few hours.

I agree that we should update the figma eventually, though if this isn't a requirement maybe we shouldn't bother? :P

File: app/src/main/java/xyz/kotlout/kotlout/view/fragment/ExperimentListFragment.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    ALL,
+    SUBSCRIBED
+  }
+
+  private ListType type;
+
+  public class ExperimentListAdapter extends BaseExpandableListAdapter {
+
+    private String[] groups = {"Open experiments", "Closed Experiments"};
+    private String[][] children = {{"peepee"}, {"Poopoo"}};
```
Comment (dlallan) [inline app/src/main/java/xyz/kotlout/kotlout/view/fragment/ExperimentListFragment.java:30]: Classy

**Review by dlallan — APPROVED** [review]

**Comment by Mia2b** [comment]
Looks good!
As [Tharidu said](https://github.com/CMPUT301W21T02/Kotlout/pull/37#pullrequestreview-609256493) as long as we make the story board or this to match at some point it should be fine. I do think the Subscription heart should be changed to solid fill to match the other icons, the purely cosmetic though.

**Pull request merged**